### PR TITLE
Add node binaries to the release notes.

### PR DESCRIPTION
**Pull request by brendandburns**
Also clean up some shell lint, this file does *not* obey bash
style guide :P

@saad-ali 

Fixes https://github.com/kubernetes/release/issues/261

### CODE REVIEW

**Comment by saad-ali** [comment]
/lgtm

+ @ixdy @david-mcmahon for approval

File: relnotes [hunk 1]
```diff
@@ -225,14 +225,15 @@ create_body () {
   echo "[Documentation](https://docs.k8s.io) &" \
        "[Examples](https://releases.k8s.io/$CURRENT_BRANCH/examples)"
   echo
-  if [[ -n $release_tars ]]; then
-    echo "## Downloads for $title"
+  if [[ -n ${release_tars} ]]; then
```
Comment (ixdy) [inline relnotes]: this file doesn't follow the bash style guide, but please revert these changes for consistency.

**Comment by ixdy** [comment]
ping

**Comment by brendandburns** [comment]
@ixdy sorry for the delay. addressed.
